src/index.html

```diff
@@ -1,14 +1,131 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8"></meta>
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    ></meta>
+    <meta http-equiv="X-UA-Compatible" content="ie=edge"></meta>
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin = "anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Moyo header</h1>
+  <body class="body">
+    <header
+    class="header"
+    >
+      <a
+      href="#home"
+      class="logo"
+      >
+        <img
+        src="./images/logo.png"
+        alt="logo"
+        >
+      </a>
```

Comment:
```suggestion
    <header
      class="header"
    >
      <a
        href="#home"
        class="logo"
      >
        <img
          src="./images/logo.png"
          alt="logo"
        >
      </a>
```
Let's work on code style. Fix it everywhere please.